### PR TITLE
matter: fix weather station after recent FEM changes

### PR DESCRIPTION
**Pull request by Damian-Nordic**
Thingy:53 Matter weather station has BLE stability issues
after merging #6733. Linux BLE tools can't connect to the
device. Perhaps, the issue occurs only on older revisions
of Thingy:53.

Signed-off-by: Damian Krolik <damian.krolik@nordicsemi.no>

### CODE REVIEW

File: applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/child_image/multiprotocol_rpmsg.conf [hunk 1]
```diff
@@ -15,3 +15,6 @@ CONFIG_BT_MAX_CONN=2
 CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
 CONFIG_BT_BUF_ACL_TX_SIZE=251
 CONFIG_BT_BUF_ACL_RX_SIZE=502
+
+# Workaround FEM issue seen on older Thingy:53 revisions
+CONFIG_MPSL_FEM_NRF21540_TX_GAIN_DB_POUTA=10
```
Comment (kkasperczyk-no) [inline applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/child_image/multiprotocol_rpmsg.conf:20]: Does it affect only our application or other Thingy:53 applications should be aligned too?

Comment (Damian-Nordic) [reply applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/child_image/multiprotocol_rpmsg.conf:20]: The thing is the default setting is supposedly correct and the problem is that we use too old Thingy:53 revisions, so I don't want to take the responsibility of changing it for all teams.

Comment (kkasperczyk-no) [reply applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/child_image/multiprotocol_rpmsg.conf:20]: Ok and what if we will use the other applications with our old Thingys in CI to verify DFU? Does it mean they may fail to connect too?

Comment (Damian-Nordic) [reply applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/child_image/multiprotocol_rpmsg.conf:20]: I think then we can change this flag in our test. I think other applications may also struggle from that depending on which Thingy revision they are run on.

**Review by kkasperczyk-no — APPROVED** [review]

**Review by LuDuda — APPROVED** [review]

**Pull request merged**